File: as-2020-ui-master/src/model/game-parameters.ts
import { Observable } from 'rxjs';
import { toObservable } from './util';
import Axios from 'axios';
import { BASE_URL } from '../constants';
import { map } from 'rxjs/operators';

export interface GameParameters {
  batchSize: number;
  baseScriptRate: number;
  itemStepRate: number;
  coffeeDrinkDuration: number;
  energizedDuration: number;
  watchTvDuration: number;
  entertainedDuration: number;
  baseScoreRates: { [eventName: string]: number };
  baseScorePerTick: { [eventName: string]: number };
}

export const getGameParameters = (): Observable<GameParameters> => {
  return toObservable(
    Axios.get<GameParameters>(BASE_URL + 'game/parameters/')
  ).pipe(map(response => response.data));
};

File: as-2020-api-master/AS2020Api/Model/GameParameters.cs
using System.Collections.Generic;
using System.Linq;

namespace AS2020Api.Model
{
    public class GameParameters
    {
        public readonly int BatchSize = 5;
        public readonly int BaseScriptRate = 1;
        public readonly int ItemStepRate = 10;
        public readonly int CoffeeDrinkDuration = 4;
        public readonly int EnergizedDuration = 10;
        public readonly int WatchTvDuration = 6;
        public readonly int EntertainedDuration = 30;

        public readonly IDictionary<PlayEventType, int> BaseScoreRates =
            PlayEventProperties.GetAll().ToDictionary(p => p.Type, p => p.BaseEventScore);

        public readonly IDictionary<PlayEventType, int> BaseScorePerTick =
            PlayEventProperties.GetAll().ToDictionary(p => p.Type, p => p.BaseTickScore);
    }
}
